public/app/features/plugins/sandbox/sandbox_components.tsx

```diff
@@ -70,7 +70,8 @@ export async function sandboxPluginComponents(
 
   // config pages
   if (Reflect.has(pluginObject, 'configPages')) {
-    const configPages: Record<string, PluginConfigPage<any>> = Reflect.get(pluginObject, 'configPages');
+    const configPages: NonNullable<GrafanaPlugin['configPages']> = Reflect.get(pluginObject, 'configPages') ?? [];
+
```

Comment:
@grafana/plugins-platform-frontend plz review this 🙏

Reply:
These changes look correct for the sandbox code